pydata/urls.py

```diff
@@ -0,0 +1,14 @@
+from django.conf.urls import url
+
+from . import views
+
+
+urlpatterns = [
+    url(r'persons/add/?$', views.PersonCreate.as_view(), name='person_add'),
+    url(r'persons/import/?$', views.PersonImport.as_view(), name='person_import'),
+
+    url(r'^events/import/?$', views.ConferenceImport.as_view(), name='event_import'),
+
+    url(r'^tasks/add/?$', views.TaskCreate.as_view(), name='task_add'),
+    url(r'^tasks/import/?$', views.TaskImport.as_view(), name='task_import'),
+]
```

Comment:
I was just thinking if you really need to plug into existing views. Maybe you should have one huge view that imports everything from PyData instead?